client/flutter/lib/components/carousel/carousel_slide.dart

```diff
@@ -87,6 +98,9 @@ class _CarouselSlideState extends State<CarouselSlide> {
           ],
         ),
         onTap: () {
+          widget.analytics.logEvent(
+              name: 'CarouselSlideLearnMore',
+              parameters: {'title': AnalyticsUtil.cleanValue(widget.title)});
```

Comment:
This will change based on localization.  You can pass in analyticsTag or index instead

Reply:
> This will change based on localization. You can pass in analyticsTag or index instead

Done, now using index: 0d87f3bcc74ff3f9745f26853670cb06cc8cd1f5